R/mobr.R

```diff
@@ -473,6 +503,18 @@ get_plot_dens = function(comm, density_stat){
     return(plot_dens)   
 }
 
+#' Auxillary function for effect_ functions
+#' Compute an overall p-value for one factor in the discrete case
+#' p-value is based on mean squared difference from zero summed across the scales
+#' @keywords internal
```

Comment:
It's slightly different I think but my brain is too foggy right now to remember what I did and why... Can I get back to you during the weekend?